app/src/main/java/org/mozilla/fenix/home/recenttabs/controller/RecentTabController.kt

```diff
@@ -62,6 +67,11 @@ class DefaultRecentTabsController(
         navController.navigate(HomeFragmentDirections.actionGlobalTabsTrayFragment())
     }
 
+    override fun handleRecentSearchGroupClicked(tabId: String) {
+        selectTabUseCase.invoke(tabId)
```

Comment:
I didn't think about selecting the tab first to get the scrollToPosition for it, but this could work. 🙂